Sources/UI/ActionButton/ActionButton.swift

```diff
@@ -8,22 +8,30 @@
 
 import SwiftUI
 
-public class ActionButtonViewModel: ObservableObject {
-  @Published public var title: String = ""
-  @Published public var font: Font = .system(size: 18)
+@available(iOS 13, *)
+private class ActionButtonViewModel: ObservableObject {
+  @Published public var title: String = "Button"
+  @Published public var font: Font = .system(size: 16)
   @Published public var textColor: Color = .white
-  @Published public var cornerRadius: ActionButton.CornerRadiusType = .custom(0)
+  @Published public var cornerRadius: ActionButton.CornerRadiusType = .rounded
   @Published public var backgroundType = ActionButton.Background.plain(.blue)
   @Published public var borderColor: Color = .clear
   @Published public var borderWidth: CGFloat = 0
-  @Published public var extraImage: ActionButton.ExtraImage?
+  @Published public var leftImage: ActionButton.ExtraImage?
+  @Published public var rightImage: ActionButton.ExtraImage?
+  @Published public var titleLeftImage: ActionButton.ExtraImage?
+  @Published public var titleRightImage: ActionButton.ExtraImage?
+  @Published public var actionHandler: (() -> Void)?
```

Comment:
This used to be public when we accessed the properties directly, now since we have made the access through builder methods we don't need it as this anymore, but forgot to move the definition inside the ActionButton.  Nice catch!